web_src/css/base.css

```
@@ -525,6 +526,10 @@ a.label,
   line-height: var(--line-height-default); /* fomantic uses "1" which causes overflow problems because "1" doesn't consider the descent part */
 }
 
+#navbar .item {
```

```suggestion
#navbar > .menu > .item {
```

Otherwise it will also match dropdown items from `#navbar .dropdown .item`.

---

Another question is: why is this needed?

If the users would like to customize the colors, they could just put the following code in their CSS, but not introducing a new variable. Otherwise, there would be too many variables for such "customization", eg: the footer?

```css
#navbar > .menu > .item {
    color: ...
}
```

---

I guess having those vars ensures some kind of stability for themes, which I think is a plus. But on the other hand a variable that's only used once could be seen as borderline justified.

---

My idea was to simplify theming ... but I go with eiher decissions

---

And yes have stable api upstream :)

---

That's a valid point.
Approving under that circumstance.

---

> That's a valid point. Approving under that circumstance.

The selector is still not right yet